packages/store-indexer/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@latticexyz/store-indexer",
+  "version": "2.0.0-next.0",
+  "description": "Minimal Typescript indexer for Store",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/latticexyz/mud.git",
+    "directory": "packages/store-indexer"
+  },
+  "license": "MIT",
+  "type": "module",
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "types": "src/index.ts",
+  "scripts": {
+    "build": "pnpm run build:js",
+    "build:js": "tsup",
+    "clean": "pnpm run clean:js",
+    "clean:js": "rimraf dist",
+    "dev": "tsup --watch",
+    "lint": "eslint .",
+    "start": "tsx bin/sqlite-indexer",
+    "start:local": "SQLITE_FILENAME=anvil.db CHAIN_ID=31337 pnpm start",
+    "start:testnet": "SQLITE_FILENAME=testnet.db CHAIN_ID=4242 START_BLOCK=19037160 pnpm start",
+    "start:testnet2": "SQLITE_FILENAME=testnet2.db CHAIN_ID=4243 pnpm start",
+    "test": "tsc --noEmit --skipLibCheck"
```

Comment:
We should remove the `skipLibCheck`  flag once https://github.com/drizzle-team/drizzle-orm/issues/682 is fixed.